lib/improver/constants.py

```diff
@@ -29,12 +29,6 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 """Module to contain generally useful constants."""
-from iris.coords import AuxCoord
-
-
-def make_constant_with_units(constant, units):
```

Comment:
This historic reason for this functionality is because when you divide or multiple a cube by a coordinate like this, Iris takes care of the units for you, which is especially useful if you have your cube in a different unit. However it seems like we never used it so I don't mind you getting rid of it.